frontend/src/utils/getEventEmitter.ts

```diff
@@ -0,0 +1,4 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
this should be under the graph manager not a global one

Reply:
Okay